src/main/java/org/example/jvspringbootfirstbook/controller/BookController.java

```diff
@@ -43,14 +45,16 @@ public BookDto getBookById(@PathVariable Long id) {
     }
```

Comment:
I think you are missing this requirement:
For users with the USER role:
`GET: /api/books`;
`GET: /api/books/{id}`.